airbyte-integrations/connectors/source-insightly/icon.svg

```diff
@@ -1 +0,0 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="250" height="250" fill="none"><path fill="#FF5621" d="m204.648 193.714.047.177c-6.087-.791-9.532-7.065-9.532-24.255 0-10.023 1.265-19.599 2.501-28.957 1.182-8.94 2.336-17.681 2.336-26.421 0-16.75-4.837-30.098-23.581-30.098-16.63 0-34.771 18.74-41.726 33.505.894-5.736 2.376-10.645 3.649-14.863 1.31-4.341 2.399-7.95 2.399-10.976a8.898 8.898 0 0 0-2.422-7.594 8.987 8.987 0 0 0-7.556-2.63c-9.676 0-31.447 9.09-34.168 22.438-1.813 6.526-2.72 11.641-3.024 58.213-.139 8.025-.865 16.026-1.602 23.488a10.138 10.138 0 0 1-4.033 6.422 10.226 10.226 0 0 1-7.377 1.852c-6.96 0-10.292-9.095-10.292-38.383 0-18.698 2.84-34.268 4.892-45.523 1.162-6.373 2.072-11.362 2.072-14.755a10.72 10.72 0 0 0-.65-4.326 10.759 10.759 0 0 0-2.324-3.712 10.819 10.819 0 0 0-3.615-2.485 10.858 10.858 0 0 0-4.31-.849 41.184 41.184 0 0 0-16.575 6.62 40.966 40.966 0 0 0-12.186 12.996C33.028 112.699 30 151.937 30 174.684c0 26.156 9.688 47.197 38.147 47.197a35.364 35.364 0 0 0 22.453-7.622c.7 4.805 2.568 7.622 6.602 7.622 28.419 0 33.864-12.208 33.864-19.88 0-45.432 10.582-80.644 22.675-80.644 5.141 0 6.954 5.114 6.954 12.497 0 6.187-1.063 13.975-2.16 22.007-1.168 8.552-2.374 17.381-2.374 24.849 0 25.555 13.303 41.171 37.491 41.171a35.943 35.943 0 0 0 24.017-8.584c3.129-4.779 2.579-8.77 1.636-13.937-.867-4.751-9.775-5.739-14.657-5.646ZM72.733 26.88c-10.549 0-28.34 8.22-28.34 25.81 0 19.856 8.75 23.188 17.489 23.189A31.792 31.792 0 0 0 83.28 64.933a31.509 31.509 0 0 0 7.541-22.744c0-11.058-8.435-15.31-18.089-15.31Z"/></svg>
```

Comment:
Why you removed this file?

Reply:
Must have been an accident. I have added it back now